app/src/main/res/layout/o_event_qrcode_view.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can you rename the file to use "organiser" prefix instead of "o"

Reply:
I went with that since there a some other files using that convention.